libr/anal/p/anal_arc.c

```diff
@@ -1092,6 +1092,6 @@ struct r_anal_plugin_t r_anal_plugin_arc = {
 struct r_lib_struct_t radare_plugin = {
 	.type = R_LIB_TYPE_ANAL,
 	.data = &r_anal_plugin_arc,
-	.version = R2_VERSION
+	.version = R2_VERSION,
```

Comment:
The trailing comma is C99 compliant (and I'm sure I remember it working in GCC before that), but I guess I'm not fully aware of all the compilers that are being supported...